### PR TITLE
WIP count optimization

### PR DESCRIPTION
**Pull request by tgruben**
## Overview
by circumventing the generic container iteration, i'm able to improve
 the speed  TestBitmap_Quick_LargeValue by 4x

### CODE REVIEW

File: roaring/roaring.go [hunk 1]
```diff
@@ -93,6 +93,7 @@ type Containers interface {
 	// return the first container at or after key. found will be true if a
 	// container is found at key.
 	Iterator(key uint64) (citer ContainerIterator, found bool)
+	Count() uint64
```
Comment (travisturner) [inline roaring/roaring.go]: At first I was a little confused by the name of this method; "does this count the number of containers?". But I suppose as long as the comment specifies that `Count returns the number of bits set in the containers.` then it's ok.

Comment (travisturner) [reply roaring/roaring.go]: I've been trying to figure out which fields we made available for the b+tree stuff.
It looks like that's in commit 4282e90f where we added a `Mapped()` getter method and an `Update()` method. Would adding a `N()` getter method work for you?

Comment (tgruben) [reply roaring/roaring.go]: Works for me, like it @codysoyland ?

Comment (codysoyland) [reply roaring/roaring.go]: Seems worth the export 👍

**Pull request merged**